bentoml/__init__.py

```diff
@@ -17,12 +17,15 @@
 __version__ = get_versions()['version']
 del get_versions
 
-from bentoml.configuration import config
+from bentoml.configuration import config, inject_dependencies
 from bentoml.utils.log import configure_logging
 
 # Configuring logging properly before loading other modules
 configure_logging()
 
+# Inject dependencies and configurations
+inject_dependencies()
```

Comment:
Is this intentionally behind the `configure_logging` call? I'm curious what would happen if we want to inject configs in the logging module in the future

Reply:
It is intentional for now because `inject_dependenies` has some logging statements.